pkg/lib/machine-dialogs.js

```diff
@@ -428,26 +480,36 @@ function AddMachine(dialog) {
         if (!limit || isNaN(limit))
             limit = 20;
 
-        dialog.render({
-            nearlimit : limit * 0.75 <= dialog.machines_ins.list.length,
-            limit : limit,
-            placeholder : _("Enter IP address or host name"),
-            options : invisible,
-        });
+        self.user = { };
 
-        var button = dialog.get_sel(".modal-footer>.pf-m-primary");
-        button.on("click", add_machine);
+        cockpit.user()
+                .done(function (user) {
+                    self.user = user;
+                })
+                .always(function (user) {
+                    dialog.render({
+                        nearlimit : limit * 0.75 <= dialog.machines_ins.list.length,
+                        limit : limit,
+                        placeholder : _("Enter IP address or host name"),
+                        default_user : self.user.name,
+                        options : invisible,
+                    });
 
-        $("#add-machine-address").on("keyup", function (ev) {
-            if (ev.which === 13) {
-                var disabled = button.prop('disabled');
-                if (!disabled)
-                    add_machine();
-            }
-        });
-        $("#add-machine-address").on("input focus", check_address);
-        color_picker.render("#add-machine-color-picker", dialog.address, color);
-        $("#add-machine-address").focus();
+                    var button = dialog.get_sel(".modal-footer>.pf-m-primary");
+                    button.on("click", add_machine);
+
+                    $("#add-machine-address").on("keyup", function (ev) {
+                        if (ev.which === 13) {
+                            var disabled = button.prop('disabled');
+                            if (!disabled)
+                                add_machine();
+                        }
+                    });
+                    $("#add-machine-address").on("input focus change", check_address);
+                    $("#add-machine-user").on("input", function () { user_name_dirty = true });
```

Comment:
I don't know if this is the correct line, but the placeholder text above and the "current user" both being placeholder texts is confusing. One explains what kind of input you can use, the second one is actually a default value the dialog would accept. I think the username should not be placeholder text.

![Screenshot from 2020-08-18 13-58-28](https://user-images.githubusercontent.com/209895/90510360-21966780-e15b-11ea-889c-92b9e99f26c5.png)